Makefile

```diff
@@ -245,21 +276,32 @@ test_redirections_w_zsh: build
 test_completions_w_zsh: build
 	$(call run_test,$(COMPLETIONS_STAGES_ZSH),zsh)
 
+test_history_w_bash: build
+	$(call run_test,$(HISTORY_STAGES),bash)
```

Comment:
@Arpan-206 we also need to edit `stages_test.sh` so that we get actual fixtures created. These make commands are only for local testing, they don't actually get run on CI